### ISSUE TITLE
Error "LEDC is not initialized" introduced in commit 4ce2435 - Fix Bluetooth LED during WiFi config

### ISSUE THREAD

**Issue by doug-foster** [sparkfun/SparkFun_RTK_Everywhere_Firmware#287]
### Subject of the issue
Error "LEDC is not initialized" introduced in commit 4ce24351ebf7e546a7430a0dd6701b028e132549 - Fix Bluetooth LED during WiFi config

### Your workbench
* Latest version of RTK firmware 1.1 @ Commit e99a7daf927b8fa154a6dd41268e5a17472e367f
* EVK version 1.1 (same for EVK version 1.0)

### Steps to reproduce
1. Roll back to Commit "67907f01ac49d2c900790d2c911656e9a8ac0113 - Fix radio tooltip", build & upload image - no error messages on CONFIG ESP32 USB port
2. Advance to next Commit "4ce24351ebf7e546a7430a0dd6701b028e132549 - Fix Bluetooth LED during WiFi config", build & upload image - errors on CONFIG ESP32 USB port
3. Error still exists in latest version of RTK firmware 1.1 @ Commit e99a7daf927b8fa154a6dd41268e5a17472e367f in RC branch

### Expected behavior
no error messages on CONFIG ESP32 USB port

### Actual behavior
Bluetooth SPP and BLE broadcasting as: EVK Rover-8ECE
E (15729) ledc: ledc_set_duty(720): LEDC is not initialized
E (15729) ledc: ledc_update_duty(648): LEDC is not initialized
STATE_ROVER_NOT_STARTED --> STATE_ROVER_NO_FIX, 2024-05-15 22:09:10.732
Rover Accuracy (m): 0.208, SIV: 29 GNSS State: 3D Fix
STATE_ROVER_NO_FIX --> STATE_ROVER_FIX, 2024-05-15 22:09:10.809
E (16229) ledc: ledc_set_duty(720): LEDC is not initialized
E (16229) ledc: ledc_update_duty(648): LEDC is not initialized
E (16729) ledc: ledc_set_duty(720): LEDC is not initialized
E (16729) ledc: ledc_update_duty(648): LEDC is not initialized
E (17229) ledc: ledc_set_duty(720): LEDC is not initialized
E (17229) ledc: ledc_update_duty(648): LEDC is not initialized
E (17729) ledc: ledc_set_duty(720): LEDC is not initialized
E (17729) ledc: ledc_update_duty(648): LEDC is not initialized
Rover Accuracy (m): 0.231, SIV: 29 GNSS State: 3D Fix
E (18229) ledc: ledc_set_duty(720): LEDC is not initialized
E (18229) ledc: ledc_update_duty(648): LEDC is not initialized
E (18729) ledc: ledc_set_duty(720): LEDC is not initialized
E (18729) ledc: ledc_update_duty(648): LEDC is not initialized
E (19229) ledc: ledc_set_duty(720): LEDC is not initialized
E (19229) ledc: ledc_update_duty(648): LEDC is not initialized
E (19729) ledc: ledc_set_duty(720): LEDC is not initialized
E (19729) ledc: ledc_update_duty(648): LEDC is not initialized
Rover Accuracy (m): 0.226, SIV: 

### Discussion
* If I comment out lines 358-359 in Bluetooth.ino for latest RC version of RTK firmware 1.1, then build & upload, I see no error messages on CONFIG ESP32 USB port
358 - // bluetoothLedTask.detach(); // Reset BT LED blinker task rate to 2Hz
359 - // bluetoothLedTask.attach(bluetoothLedTaskPace2Hz, tickerBluetoothLedUpdate); // Rate in seconds, callback
* I Googled some references to similar error messages, was this maybe caused when the above lines were moved from Begin.ino to Bluetooth.ino?
* I wish I knew enough to try fixing it & do a PR, but I don't understand all the RTK_Everywhere task dependencies yet.
<img width="1385" alt="Screenshot 2024-05-15 at 5 48 11 PM" src="https://github.com/sparkfun/SparkFun_RTK_Everywhere_Firmware/assets/8713605/6b7c6ebd-e069-4e27-86bf-9f2033756199">
<img width="1380" alt="Screenshot 2024-05-15 at 5 48 59 PM" src="https://github.com/sparkfun/SparkFun_RTK_Everywhere_Firmware/assets/8713605/2ed09ab6-e911-4b76-b793-8d6a4ae4365d">
<img width="1395" alt="Screenshot 2024-05-15 at 5 57 21 PM" src="https://github.com/sparkfun/SparkFun_RTK_Everywhere_Firmware/assets/8713605/88b43b2d-3614-4e4e-bf00-c7bf6c1f0908">

**Comment by PaulZC** [comment]
Fixed in this commit: https://github.com/sparkfun/SparkFun_RTK_Everywhere_Firmware/commit/91ca899c7e60617ddd52915f2f0a18ec5e3abfac

TL;DR: The new ```bluetoothLedTask``` code in ```bluetoothStart``` was missing the ```PIN_UNDEFINED``` guard. Nathan wouldn't see this on Torch. It appeared on EVK because there is no ```pin_bluetoothStatusLED```.
